### PR TITLE
Update KNoT Cloud CLI and stack

### PR DESCRIPTION
**Pull request by esdrasjnr**
What does this implement/fix? Explain your changes.
---------------------------------------------------
This patch updates the CLI to create user or app tokens and change stacks to configure authn service

Where has this been tested?
---------------------------
**Operating System/Platform:** Ubuntu 18.04
**NPM Version:** 6.14
**NodeJS Version:** 12.15

### CODE REVIEW

File: src/cmds/cloud/createAppToken.js [hunk 1]
```diff
@@ -0,0 +1,54 @@
+/* eslint-disable no-console */
+import yargs from 'yargs';
+import chalk from 'chalk';
+import { Authenticator } from '@cesarbr/knot-cloud-sdk-js';
+
+const createAppToken = async (args) => {
```
Comment (netoax) [inline src/cmds/cloud/createAppToken.js]: Wouldn't it be better just update the existing `createToken` operation to support creating 'app' tokens?

**Review by netoax — CHANGES_REQUESTED** [review]

File: src/cmds/cloud/createAppToken.js (hunk 1 above)
Comment (esdrasjnr) [reply src/cmds/cloud/createAppToken.js]: Done!

File: src/cmds/cloud/createUser.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ const createUser = async (args) => {
 
   try {
     await client.createUser(args.email, args.password);
-    const token = await client.createToken(args.email, args.password);
+    const token = await client.createToken(args.email, args.password, 'user');
```
Comment (netoax) [inline src/cmds/cloud/createUser.js:15]:
```suggestion
    const token = await client.createToken(args.email, args.password, args.type);
```

**Review by netoax — CHANGES_REQUESTED** [review]

**Review by netoax — APPROVED** [review]
LGTM!

**Pull request merged**